lxc/main_aliases.go

```diff
@@ -49,6 +49,8 @@ func findAlias(aliases map[string]string, origArgs []string) ([]string, []string
 }
 
 func expandAlias(conf *config.Config, args []string) ([]string, bool, error) {
+	var completion = false
+	var completionFrament string
```

Comment:
`completeFragment`?

Reply:
For something like this, should I push a separate commit to fix the typo or modify the cherry-picked commit?

Reply:
I think just modify the cherry-picked commit.